.github/workflows/create-release-tag.yml

```
@@ -68,31 +68,20 @@ jobs:
-
-Previous version: ${LAST_TAG}
-Release branch: ${GITHUB_REF_NAME}
-Commit: ${GITHUB_SHA:0:7}
-Triggered by: @${GITHUB_ACTOR}
-
-This tag will trigger the automated release workflow.
-EOF
-)
+        # Create detailed tag message
+        TAG_MESSAGE="Release ${VERSION}\\n\\nCreated from PR #${PR_NUMBER}: ${PR_TITLE}\\n\\nPrevious version: ${LAST_TAG}\\nRelease branch: ${GITHUB_REF_NAME}\\nCommit: ${GITHUB_SHA:0:7}\\nTriggered by: @${GITHUB_ACTOR}\\n\\nThis tag will trigger the automated release workflow."
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)